Documentation/gettingstarted/alibabacloud-eni.rst

```diff
@@ -0,0 +1,179 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_alibabacloud_eni:
+
+*************************************************
+Setting Up Cilium in AlibabaCloud ENI Mode (beta)
+*************************************************
+
+.. note::
+
+   This is a beta feature. Please provide feedback and file a GitHub issue if you experience any problems.
+
+   The AlibabaCloud ENI integration is still subject to some limitations. See
+   :ref:`eni_limitations` for details.
+
+Create a Cluster on AlibabaCloud
+================================
+
+Setup a Kubernetes on AlibabaCloud. You can use any method you prefer.
+The quickest way is to create an ACK (Alibaba Cloud Container Service for
+Kubernetes) cluster and to replace the CNI plugin with Cilium.
+For more details on how to set up an ACK cluster please follow
+the `official documentation <https://www.alibabacloud.com/help/doc-detail/86745.htm>`_.
+
+
+Disable ACK CNI (ACK Only)
+==========================
+
+If you are running an ACK cluster, you should delete the ACK CNI.
+
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Cilium will manage ENIs instead of the ACK CNI, so any running DaemonSet from
+the list below has to be deleted to prevent conflicts.
+
+- ``kube-flannel-ds``
+- ``terway``
+- ``terway-eni``
+- ``terway-eniip``
+
+.. note::
+
+    If you are using ACK with Flannel (DaemonSet ``kube-flannel-ds``),
+    the Cloud Controller Manager (CCM) will create route (Pod CIDR) in VPC.
+    If your cluster is an Managed Kubernetes you cannot disable this behavior.
+    Please consider creating a new cluster.
+
+.. code-block:: shell-session
+
+   kubectl -n kube-system delete daemonset <terway>
+
+
+Create AlibabaCloud Secrets
+===========================
+
+Before installing Cilium, a new Kubernetes Secret with the AlibabaCloud Tokens needs to
+be added to your Kubernetes cluster. This Secret will allow Cilium to gather
+information from the AlibabaCloud API which is needed to implement ToGroups policies.
+
+AlibabaCloud Access Keys
+------------------------
+
+To create a new access token the `following guide can be used
+<https://www.alibabacloud.com/help/doc-detail/93691.htm>`_.
+These keys need to have certain `RAM Permissions
+<https://ram.console.aliyun.com/overview>`_:
```

Comment:
This link points to the Chinese login page. I don't mind keeping it, but is it possible to also have a link to the international/English equivalent please?

Reply:
in my test i can redirct to intertional login page other than chinese page . i think it is up to pervious usage. Let's keep it here :)